### ISSUE TITLE
Traitor notification not popping up

### ISSUE THREAD

**Issue by PixelTheOne** [Regalis11/Barotrauma#1822]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
The notification that lets you know that you're a traitor doesn't pop up for the players.

**Steps To Reproduce**
Installing mods??? Not sure
**Version**
Game version is v0.9.3.3 and my operating system is 64-bit Windows 10.

**Additional information**
The host of the server does get the pop-up, but the players don't and you can't see your traitor objective in the "tab menu". The only way to check if you're a traitor is to type "traitorlist" in the console. Also after reinstalling the game a couple of times, deleting the game files, unsubscribing all the mods and submarines, the traitor pop-up is still non-existent.

**Comment by EdusFF** [comment]
Tested with two PCs and was able to see the traitor popup on client's end

**Comment by PixelTheOne** [comment]
How am I supposed to give more info? This is basically what happens. The traitor notification does not pop up at all, for no apparent reason, even after reinstalling the game clean. No idea what are the steps to reproduce this. Maybe try this on my computer lol. I am starting to get really angry because i have no idea what the hell even happened. It was all fine and a day later it just broke... Can't even play with traitors on anymore...

**Comment by EdusFF** [comment]
Might be already fixed by something else we've fixed, have to see if the issue still persists after the next update.

**Comment by PixelTheOne** [comment]
Still persists, now the feature doesn't work for all of my friends

**Comment by rav2n** [comment]
I also observed this error. On the server where I play, traitors are set to "maybe."

**Comment by Regalis11** [comment]
This could potentially happen if you're playing in a submarine where the traitor objective can't be completed, for example if the sub doesn't have suitable cabinets for the traitor items to be placed in. If that's not the case, I'm not sure how to reproduce the issue - testing it myself the popups seemed to appear normally.

But in any case, we've done several fixes to the traitor messages that might be somehow related to the issue, so hopefully the next patch will fix this. Sorry for the trouble!

**Comment by daniels1989** [comment]
Just wanted to add my two cents to this. Got the game gifted yesterday from a friend, played a few games together + bots. Game version is 0.9.3.3, OS is Win7/64. Friend hosted, I joined. No mods.
Every time I was the traitor I only found out after I was dead, when it says "X was a traitor and they failed to complete their objectives." Not once did I get a message stating I was a traitor, nor did it appear under the Mission tab. The host did get their traitor mission numerous times though.
